pkg/openstackconfiggenerator/configmap.go

```diff
@@ -347,6 +347,23 @@ func createNetworksMap(
 	return networksMap, networkMappingList, nil
 }
 
+// IsRoleIncluded - checks if the role exists in the ConfigGenerator Roles set
+func IsRoleIncluded(roleName string, instance *ospdirectorv1beta1.OpenStackConfigGenerator) bool {
+
+	if len(instance.Spec.Roles) == 0 {
+		return true
+	} else if roleName == "ControlPlane" {
```

Comment:
should use openstackcontrolplane.Role const [1] to not have to hard code the string.

[1] https://github.com/openstack-k8s-operators/osp-director-operator/blob/9a77ee2c7abe10c11f20622ccdbc3a858952226f/pkg/controlplane/const.go#L30

Reply:
done